docs/RunningOnDeviceWindows.md

```diff
@@ -77,4 +77,12 @@ react-native bundle --platform windows --entry-file index.windows.js
     --dev false
 ```
 
+You will need to create the `ReactAssets` folder if it does not exist:

```

Comment:
Looks good, but please pull this up above the the bundle command since the mkdir will need to happen before that command is run.